testproject/Packages/packages-lock.json

```diff
@@ -1,11 +1,11 @@
 {
   "dependencies": {
     "com.unity.burst": {
-      "version": "1.3.2",
-      "depth": 3,
+      "version": "1.4.1",
+      "depth": 2,
       "source": "registry",
       "dependencies": {
-        "com.unity.mathematics": "1.1.0"
+        "com.unity.mathematics": "1.2.1"
       },
```

Comment:
were these upgrades necessary or just sneaked in?

Reply:
I didn't upgrade anything on purpose. This is just me still learning Unity and not realizing I shouldn't commit that file.

Reply:
Done.